src/Blocks/BlockOre.h

```diff
@@ -3,15 +3,76 @@
 
 #include "BlockHandler.h"
 
-
-
-
-
 class cBlockOreHandler:
 	public cBlockHandler
 {
 	using Super = cBlockHandler;
 
+private:
+	/** Returns the correct drop multiplier for ores
+	EXCLUDING REDSTONE ORE which is handled differently
+	works for coal, diamond, emerald, nether gold, nether quartz and lapis
+	https://minecraft.gamepedia.com/Fortune#Ore */
+	inline unsigned int FortuneDropMult(unsigned int a_fortuneLevel)
+	{
+
+		auto & random = GetRandomProvider();
+		float RandUnif = random.RandReal();
+		switch (a_fortuneLevel)
+		{
+			case 0:
+				return 1;
+			case 1:
+				if (RandUnif<0.66)
+				{
+					return 1;
+				}
+				else
+				{
+					return 2;
+				}
+				break;
+			case 2:
+				if (RandUnif<0.5)
+				{
+					return 1;
+				}
+				else if (RandUnif<0.75)
+				{
+					return 2;
+				}
+				else
+				{
+					return 3;
+				}
+				break;
+			case 3:
+			{
+				if (RandUnif < 0.4)
+				{
+					return 1;
+				}
+				else if (RandUnif < 0.6)
+				{
+					return 2;
+				}
+				else if (RandUnif < 0.8)
+				{
+					return 3;
+				}
+				else
+				{
+					return 4;
+				}
+			}
+		}
```

Comment:
This switch statement is equivalent to:
```cpp
std::max(1, FloorC(RandUnif * (a_FortuneLevel + 2)));
```

Reply:
the switch returned a 1 multiplier for any level greater than 3; we may still want some limit to avoid high fortune levels crashing the server with a lot of pickups. std::clamp(1, 10, ...) maybe?